### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by Samarth-5109**
Screenshot (46)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (dubeyaditya29) [inline .vscode/settings.json:2]: The live server file is also getting committed!

File: styles/productListing.css [hunk 2]
```diff
@@ -3,6 +3,10 @@
 .listing-container {
   display: flex;
 }
+
+.clear {
+  margin-left: 2rem;
+}
 .sidebar {
   margin-top: 2rem;
```
Comment (dubeyaditya29) [inline styles/productListing.css:11]: you can use a single margin attribute here to give values to all directions.

**Review by dubeyaditya29 — CHANGES_REQUESTED** [review]
Overall code looks clean have given some suggestion. Looks good to merge.

**Review by JowelTisso — APPROVED** [review]
Looks Good.

File: .vscode/settings.json (hunk 1 above)
Comment (Samarth-5109) [reply .vscode/settings.json:2]: will it create any problem? how is it happening?

Comment (dubeyaditya29) [reply .vscode/settings.json:2]: I dont think it is creating any impact but we can gitignore these files as they are creating a separate folder and will not look good thinking as overall project structure. I faced the similar issue then @hamza pointed this out..

**Pull request merged**